examples/recipe-sourcing-contentful/src/components/seo.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * SEO component that queries for data with
+ *  Gatsby's useStaticQuery React hook
+ *
+ * See: https://www.gatsbyjs.org/docs/use-static-query/
+ */
+
+import React from "react"
+import PropTypes from "prop-types"
+import Helmet from "react-helmet"
+import { useStaticQuery, graphql } from "gatsby"
+
+function SEO({ description, lang, meta, title }) {
```

Comment:
We could probably omit SEO in this example to keep it focused on Contentful-related stuff - what do you think?

Reply:
My thought was that we keep the example as close as possible to the default Gatsby starter, but if we wanted to make it even simpler we could just put the news listing on the homepage and remove the SEO stuff.

Reply:
Removed SEO and tightened up the example in https://github.com/gatsbyjs/gatsby/pull/17318/commits/6bf620110f66fc4a9f9a982c7c38f32d166316b2